### PR TITLE
community/py-opencl: update to 2017.1.1 / add python3 subpkg

### PR DESCRIPTION
**Pull request by itoffshore**
community/py-opencl: update to `2017.1.1` / add `python3` subpkg

### CODE REVIEW

File: community/py-opencl/APKBUILD [hunk 1]
```diff
@@ -2,31 +2,55 @@
...
 pkgdesc="Python wrapper for OpenCL"
 url="https://pypi.python.org/pypi/pyopencl"
 arch="all !ppc64le !s390x"
 license="MIT"
-depends="python2"
-depends_dev="$pkgname"
-makedepends="python2-dev py-setuptools py-mako py-numpy ctags libffi-dev mesa-dev opencl-headers opencl-icd-loader-dev"
-subpackages="$pkgname-dev"
+depends="py-mako"
+makedepends="python2-dev python3-dev py-setuptools py-mako py-numpy ctags libffi-dev mesa-dev opencl-headers \
```
Comment (Shizmob) [inline community/py-opencl/APKBUILD]: The \ should not be needed here.

File: community/py-opencl/APKBUILD [hunk 2]
```diff
@@ -2,31 +2,55 @@
...
-	python2 configure.py --cl-enable-gl --python-exe=python2 || return 1
-	make || return 1
-	python2 setup.py test
+	python2 setup.py build
+	python3 setup.py build
 }
 
 package() {
+	mkdir -p "$pkgdir"
+	mkdir -p "$pkgdir-dev"
```
Comment (Shizmob) [inline community/py-opencl/APKBUILD]: I don't think this should be necessary? If it is, `default_dev` is doing nothing which means your `-dev` subpackage is empty anyway.

File: community/py-opencl/APKBUILD [hunk 3]
```diff
@@ -2,31 +2,55 @@
...
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+check() {
 	cd "$builddir"
-	python2 setup.py install --prefix=/usr --root="$pkgdir" --optimize=1 --skip-build || return 1
+	python2 setup.py test
```
Comment (Shizmob) [inline community/py-opencl/APKBUILD]: 👍

File: community/py-opencl/APKBUILD (hunk 1 above)
Comment (itoffshore) [inline community/py-opencl/APKBUILD]: removed

File: community/py-opencl/APKBUILD (hunk 2 above)
Comment (itoffshore) [inline community/py-opencl/APKBUILD]: yes it was empty - removed

**Pull request closed without merging**